version_plone52.cfg

```diff
@@ -0,0 +1,8 @@
+[buildout]
+
+extends =
+    http://dist.plone.org/release/5.2-latest/versions.cfg
+    buildout.cfg
+
+[versions]
+plone.app.controlpanel = 3.0.4
```

Comment:
Plone 5.2 uses plone.app.controlpanel version 4, while 5.1 uses 3.0.4. Please remove this pin.

Reply:
Remove the pin from the version_plone52.cfg file.